app/src/main/java/com/example/kitchenflow/ui/main/MainViewModel.kt

```diff
@@ -1,16 +1,26 @@
 package com.example.kitchenflow.ui.main
 
+import android.icu.util.Calendar
+import androidx.databinding.ObservableField
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.example.kitchenflow.data.entity.OrderModel
 import com.example.kitchenflow.data.repository.KitchenRepository
 import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
 import io.reactivex.rxjava3.schedulers.Schedulers
+import java.text.SimpleDateFormat
+import java.util.*
 
 class MainViewModel(private val repository: KitchenRepository) : ViewModel() {
     private val ordersData: MutableLiveData<List<OrderModel>> = MutableLiveData()
     val orders: LiveData<List<OrderModel>> = ordersData
+    val date: ObservableField<String> = ObservableField(
```

Comment:
Why observable if it set only once? If you are planing to update it in future put 
`SimpleDateFormat("EEE, MMMM dd", Locale.ENGLISH).format(
            Calendar.getInstance().time
        )` to function